adapters/Nend/NendAdapter/nativeads/GADMAdapterNendNativeAdLoader.m

```diff
@@ -0,0 +1,133 @@
+        if (!ad.imageUrl) {
+            GADMAdapterNendNativeAd *unifiedAd = [[GADMAdapterNendNativeAd alloc] initWithNormal:ad logo:nil image:nil];
+            [self didReceiveUnifiedNativeAd:unifiedAd];
+        } else {
+            if (imageOptions.disableImageLoading) {
+                GADNativeAdImage *adImage = [[GADNativeAdImage alloc] initWithURL:[NSURL URLWithString:ad.logoUrl] scale:1.0f];
+                GADMAdapterNendNativeAd *unifiedAd = [[GADMAdapterNendNativeAd alloc] initWithNormal:ad logo:logo image:adImage];
+                [self didReceiveUnifiedNativeAd:unifiedAd];
+            } else {
+                [ad loadAdImageWithCompletionBlock:^(UIImage *image) {
+                    if (image) {
+                        GADNativeAdImage *adImage = [[GADNativeAdImage alloc] initWithImage:image];
+                        GADMAdapterNendNativeAd *unifiedAd = [[GADMAdapterNendNativeAd alloc] initWithNormal:ad logo:logo image:adImage];
+                        [self didReceiveUnifiedNativeAd:unifiedAd];
+                    } else {
+                        NSError *imageError = [NSError
+                            errorWithDomain:kGADMAdapterNendErrorDomain
+                                       code:kGADErrorInternalError
+                                   userInfo:@{NSLocalizedDescriptionKey : @"Failed to load image assets."}];
+                        [self didFailToLoadWithError:imageError];
+                    }
+                }];
+            }
+        }
+    }];
+}
+
+- (void)fetchLogo:(NADNative *)ad disableLoading:(BOOL)disableLoading
+     imageHandler:(void (^)(GADNativeAdImage* _Nullable logo))imageHandler {
+    if (!ad.logoUrl) {
+        imageHandler(nil);
+    } else {
+        if (disableLoading) {
+            imageHandler([[GADNativeAdImage alloc] initWithURL:[NSURL URLWithString:ad.logoUrl] scale:1.0f]);
+        } else {
+            [ad loadLogoImageWithCompletionBlock:^(UIImage *logo) {
+                if (logo) {
+                    imageHandler([[GADNativeAdImage alloc] initWithImage:logo]);
+                } else {
+                    imageHandler(nil);
+                }
+            }];
+        }
+    }
+}
+
+- (void)didFailToLoadWithError:(NSError *)error {
+    [NSException raise:NSInternalInconsistencyException format:@"You must override %@ in a subclass", NSStringFromSelector(_cmd)];
+}
+
+- (void)didReceiveUnifiedNativeAd:(nonnull id<GADMediatedUnifiedNativeAd>)ad {
+    [NSException raise:NSInternalInconsistencyException format:@"You must override %@ in a subclass", NSStringFromSelector(_cmd)];
+}
+
+- (GADNativeAdImageAdLoaderOptions *)pullImageAdLoaderOptions:(NSArray *)options {
+    NSPredicate *predicate = [NSPredicate predicateWithFormat: @"class == %@", [GADNativeAdImageAdLoaderOptions class]];
+    return [[options filteredArrayUsingPredicate:predicate] firstObject];
+}
+
++ (BOOL)handlesUserImpressions {
+    return YES;
+}
+
++ (BOOL)handlesUserClicks {
+    return YES;
+}
```

Comment:
I think we can directly return `YES` in `GADMAdapterNendNativeVideoAd` and `GADMAdapterNendNativeAd` instead of using this method?

Reply:
OK. I'll change to return `YES` directly.